docs/tutorials/revocation/revocation.collection.json

```diff
@@ -70,12 +70,11 @@
 					}
 				],
 				"url": {
-					"raw": "{{API_BASE_URL}}/identifiers/{{ORGANIZATION_DID_WEB}}",
```

Comment:
I think we want this... the change should go the other direction... fixing all "resolution endpoints" to look like 

```
"{{API_BASE_URL}}/identifiers/{{ORGANIZATION_DID_WEB}}",
```

Reply:
Agreed - my apologies for acting without thinking this through. I've reverted the change.